### ISSUE TITLE
Hard to tell if someone is following you

### ISSUE THREAD

**Issue by RangerMauve** [beakerbrowser/fritter#40]
It would be nice if there was something on a persons fritter profile that indicated whether they followed you. That way it'd be more obvious if they'd be able to see posts from you.

Currently the way to work around this is to go to their "following" list and searching for yourself, which isn't the quickest solution.

**Comment by Gozala** [comment]
Would be even better if there was indicator telling you they won't see your message if you mention or try to respond to their post.

**Comment by puf17640** [comment]
Well if you can display the following list on someone's page, you might as well check if you're in that list. Is anyone up for implementing this or should I go and try?

**Comment by RangerMauve** [comment]
You might as well try! :D

Basically, the profile for each person contains their `follows` list. It should be easy enough to find the code where the person's name / picture is being rendered, and add some sort of indicator to show that the person `follows you`.

Or whatever you'd prefer.

**Comment by puf17640** [comment]
Shouldn't be that much of a problem, I'll have a look into it later today! Also working on a "round" theme for fritter because I really like round borders 😆 Any ideas or suggestions, anything you want to have? Otherwise I'll try to fit it to the design.

**Comment by RangerMauve** [comment]
@puf17640 I think a great idea would be to use libfritter directly and create your own front-end using whatever you're interested in.

Having more veriety for fritter front ends can only help the ecosystem grow and promote the idea of people owning their data and how they intereact with it.

**Comment by puf17640** [comment]
That would be cool too, but too much work for me right now, since school is keeping me pretty busy right now. I'll definitely keep that in mind and have a look into it, it might be a foundation to what I was going to build anyway because I thought about building a p2p version of youtube or any other web2.0 service, since I really like rebuilding stuff from scratch.

**Comment by puf17640** [comment]
Well I just cloned the repo and it seems to be kind of broken right now. Also it seems like fritter.hashbase.io is not up to date since the last merge commit is not included. I'll just try to get it working I guess... gonna be a great night lmao

**Comment by pfrazee** [comment]
Yeah I've stopped putting work into Fritter because it's not been performing how I want. I'm in the process of a rewrite, but it's going to be a couple months. (The rewrite will depend on some new Beaker APIs like the identity system and data protocols.)

**Comment by puf17640** [comment]
@pfrazee okay, that's understandable since it's a year old now and I can imagine how much beaker has evolved in that time. I just found out about beaker 2 days ago and I'm already fully tied in! 😆 I've been really into cryptography and peer-to-peer projects for a while now and I hope I can somehow contribute in the future! Since I'm still in school I don't have any experience worth to be mentioned but if you've got anything I'd be really happy to try and help.

**Comment by pfrazee** [comment]
@puf17640 Awesome! We keep "help wanted" tickets on the Beaker repo (https://github.com/beakerbrowser/beaker/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) and we'll keep everybody updated on the updates around tooling for apps like Fritter.
